rclcpp/topics/minimal_publisher/member_function_with_unique_network_flow_endpoints.cpp

```diff
@@ -42,18 +42,24 @@ class MinimalPublisherWithUniqueNetworkFlowEndpoints : public rclcpp::Node
 
     // Create publisher without unique network flow endpoints
     // Unique network flow endpoints are disabled in default options
-    auto options_2 = rclcpp::PublisherOptions();
     publisher_2_ = this->create_publisher<std_msgs::msg::String>("topic_2", 10);
     timer_2_ = this->create_wall_timer(
       1000ms, std::bind(&MinimalPublisherWithUniqueNetworkFlowEndpoints::timer_2_callback, this));
 
-    // Get network flow endpoints
-    auto network_flow_endpoints_1 = publisher_1_->get_network_flow_endpoints();
-    auto network_flow_endpoints_2 = publisher_2_->get_network_flow_endpoints();
+    try {
+      // Get network flow endpoints
+      auto network_flow_endpoints_1 = publisher_1_->get_network_flow_endpoints();
+      auto network_flow_endpoints_2 = publisher_2_->get_network_flow_endpoints();
 
-    // Print network flow endpoints
-    print_network_flow_endpoints(network_flow_endpoints_1);
-    print_network_flow_endpoints(network_flow_endpoints_2);
+      // Print network flow endpoints
+      print_network_flow_endpoints(network_flow_endpoints_1);
+      print_network_flow_endpoints(network_flow_endpoints_2);
+    } catch (const rclcpp::exceptions::RCLError & e) {
+      RCLCPP_ERROR(
+        this->get_logger(),
+        "Error: %s",
```

Comment:
Two more minor things here.  The first is that I think we should have a comment block at the top of the `try/catch` sequence explaining that we expect this to fail in some situations, and that is expected.

The second is that we should probably not print "Error" here, since it is not a hard error.  Maybe "Warning" or even "Info" here would be better, since this is an allowed situation.

Reply:
agree, addressed with https://github.com/ros2/examples/pull/313/commits/0bd84d886660d8ad72d272fbfa90662811aeeb5e